lib/etcd_databag.py

```diff
@@ -24,6 +24,9 @@ class EtcdDatabag:
      'cluster_address': '127.0.0.1',
      'db_address': '127.0.0.1',
      'unit_name': 'etcd0',
+     'heartbeat-interval': '100',
+     'election-timeout': '1000',
+     'snapshot-count': '10000',
```

Comment:
match the default in `config.yaml`
```suggestion
     'snapshot-count': '100000',
```